plugins/kapt3/kapt3-base/src/org/jetbrains/kotlin/kapt3/base/ProcessorLoader.kt

```diff
@@ -48,7 +51,21 @@ open class ProcessorLoader(private val options: KaptOptions, private val logger:
             logger.info { "Annotation processors: " + processors.joinToString { it::class.java.canonicalName } }
         }
 
-        return LoadedProcessors(processors, classLoader)
+        return LoadedProcessors(wrapInIncrementalProcessor(processors, classpath), classLoader)
```

Comment:
qq: should this be checking options.incrementalKapt before wrapping the processor? I'm seeing new behavior related to reading zip entries from the classpath from the getIncrementalProcessorsFromClasspath() method despite the incrementalKapt option being in its default state (off). Unsure if this is intentional or not.

Reply:
Thanks for catching this, I've uploaded https://github.com/JetBrains/kotlin/pull/2334 to fix it.